web_tree_dynamic_colored_field/static/description/index.html

```diff
@@ -515,7 +515,7 @@ <h1>Other credits</h1>
 <p>The development of this module has been financially supported by:</p>
 <ul class="simple">
 <li>Camptocamp</li>
-<li>Versada.eu</li>
+<li>Versada</li>
```

Comment:
As I said, you must not edit `index.html` for OCA modules, this is generated automatically by OCA tools. You have to modify only `rst` files which index is generated from

Reply:
hmm, I didn't -- pre-commit did :shrug: